paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -421,6 +461,17 @@ void CompareNativeAndAnalysis(
   CompareResult(analysis_outputs, native_outputs);
 }
 
+void CompareQuantizedAndAnalysis(
+    const PaddlePredictor::Config *config,
+    const PaddlePredictor::Config *qconfig,
+    const std::vector<std::vector<PaddleTensor>> &inputs) {
+  PrintConfig(config, true);
+  std::vector<PaddleTensor> analysis_outputs, quantized_outputs;
```

Comment:
Could you add some log like `LOG(INFO) << "FP32 start..."` before line470 and`LOG(INFO) << "INT8 start..."` before line471?

Reply:
ok

Reply:
Can we do that in next PR?